### PR TITLE
Added instructions for windows

### PR DESCRIPTION
**Pull request by akansha006**

### CODE REVIEW

File: nomad/settings.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = (os.environ.get('RUNNING_PROD') is None)
 
-ALLOWED_HOSTS = ['localhost', '.herokuapp.com']
+ALLOWED_HOSTS = ['localhost', '.herokuapp.com','127.0.0.1']
```
Comment (prabhakar267) [inline nomad/settings.py:28]: Doesn't `localhost` serve same purpose as `127.0.0.1 `

File: README.md [hunk 2]
```diff
@@ -31,6 +31,14 @@ $ sudo -u postgres psql
 psql=# alter user nomad with encrypted password 'pass';
 psql=# grant all privileges on database nomad to nomad ;
 ```
+For Windows-
```
Comment (prabhakar267) [inline README.md:34]: Can you check formatting as well
+ you missed a ``` somewhere
+ If you want to have a line break in output, you would have to have 2 line breaks in your markdown

**Review by prabhakar267 — CHANGES_REQUESTED** [review]

File: nomad/settings.py (hunk 1 above)
Comment (akansha006) [reply nomad/settings.py:28]: yeah...actually I missed to remove that change.

**Comment by prabhakar267** [comment]
@akansha006 can you address the other comment as well?
Squash everything into one commit

**Comment by prabhakar267** [comment]
@akansha006 changes look good, just squash them in one commit
Reference: https://stackoverflow.com/q/5189560/5602759

**Review by prabhakar267 — APPROVED** [review]

**Pull request closed without merging**